.docker/ci/docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+/bin/bash -c "find /opt/ros/crystal/ -name tf2_eigen | xargs rm -rf && source /opt/ros/crystal/setup.bash \
+	      && touch /ros2_ws/src/image_common/camera_calibration_parsers/COLCON_IGNORE \
```

Comment:
`camera_calibration_parsers` and `camera_info_manager` can be skipped with `--packages-skip camera_info_manager camera_calibration_parsers`

Reply:
I prefer to have the de `colcon` command as clean as we can, but if you prefer to skip the packages using this way, I'll change it.